prac_06/guitars.py

```diff
@@ -0,0 +1,52 @@
+"""
+Do-From-Scratch Exercises, Guitars!
+Estimate: 80 minutes
+Actual: 102 minutes
+"""
+
+from prac_06.guitar import Guitar
+
+
+def main():
+    """Get guitars referencing the guitars class and display them"""
+    guitars = []
+    print("My Guitars!")
+    guitars.append(Guitar("Gibson L-5 CES", 1922, 16035.40))
+    guitars.append(Guitar("Line 6 JTV-59", 2010, 1512.9))
+    name = input("Name: ")
+    while name != "":
+        prompt = "Year: "
+        year = get_valid_number(prompt)
+        prompt = "Cost: $"
+        cost = get_valid_number(prompt)
+        guitar = Guitar(name, int(year), cost)
+        guitars.append(guitar)
+        print(f"{guitar} added.")
+        name = input("Name: ")
+
+    display_guitars(guitars)
+
+
+def display_guitars(guitars):
```

Comment:
The sample output started with something like this :"These are my guitars:"
you may want to add it. 👍